content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the chat engine API
+---
+
```

Comment:
@Neema-2016, kindly remove the `---` tags and just have standard heading format.